pkg/scheduler/cache/cache.go

```diff
@@ -36,7 +36,7 @@ import (
 	infov1 "k8s.io/client-go/informers/core/v1"
 	schedv1 "k8s.io/client-go/informers/scheduling/v1"
 	storagev1 "k8s.io/client-go/informers/storage/v1"
-	storagev1alpha1 "k8s.io/client-go/informers/storage/v1alpha1"
```

Comment:
the "storagev1alpha1" should be changed. Additionally, is the change compatible for v1alpha1 object?

Reply:
> the "storagev1alpha1" should be changed. Additionally, is the change compatible for v1alpha1 object?

Sorry for the clerical error. In fact, `storagev1apha1` has already been changed to `storagev1beta1`. The reference: https://github.com/volcano-sh/volcano/pull/1910/files